Update React Native content (#3166)
getsentry/sentry-docs: src/includes/getting-started-install/react-native.mdx

Add the `@sentry/react-native` dependency:

```bash {tabTitle:npm}
npm install --save @sentry/react-native
```

```bash {tabTitle:Yarn}
yarn add @sentry/react-native
```

<Note>

> If you are using Expo, see [How to Add Sentry to Your Expo Project](https://docs.expo.io/guides/using-sentry/). This SDK works only for either ejected projects or projects that directly use React Native.

Reviewer: ```suggestion
If you are using Expo, see [How to Add Sentry to Your Expo Project](https://docs.expo.io/guides/using-sentry/). This SDK works for both managed and bare projects.
```
Original text is no longer true as Expo updated it so it works on all expo projects now

</Note>

### Linking

Link the SDK to your native projects to enable native crash reporting. If you're using `react-native` >=0.60:

```bash {tabTitle:NPX}
npx @sentry/wizard -i reactNative -p ios android

cd ios
pod install
```

```bash {tabTitle:Yarn}
yarn sentry-wizard -i reactNative -p ios android

cd ios
pod install
```

Next, install the dependencies for [auto-linking](https://github.com/react-native-community/cli/blob/master/docs/autolinking.md) in iOS, which relies on CocoaPods.

> If you are running a project with `react-native < 0.60`. you still need to call `react-native link`.

Reviewer: `react-native link` still needs to be done for <0.60, and needs to be before the sentry-wizard and pod install steps above

Author: Can you clarify? As this reads now we're still telling users to run the `react-native link`, with the code sample, then use Sentry wizard. So, I'm not sure what change to make, other than to move line 37 prior to line 35, perhaps?
